### PR TITLE
chore: fix linter spellcheck issues pt.2

### PR DESCRIPTION
**Pull request by evcodes**
#### Description of changes

Second part of spellcheck changes

#### Description of how you validated changes
ran yarn test, triggering e2e tests

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [n/a] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#tests)
- [n/a] Relevant documentation is changed or added (and PR referenced)
- [n/a] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [n/a] [Pull request labels](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#labels) are added

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by jhockett — CHANGES_REQUESTED** [review]

File: packages/amplify-category-custom/src/utils/constants.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ export const CFN_DEPLOYMENT_NAME = 'AWS CloudFormation';
 export const customDeploymentOptions = [CDK_DEPLOYMENT_NAME, CFN_DEPLOYMENT_NAME];
 export const categoryName = 'custom';
 export const CDK_SERVICE_NAME = 'customCDK';
-export const CFN_SERVICE_NAME = 'customCloudformation';
+export const CFN_SERVICE_NAME = 'customCloudFormation';
```
[the remark this answers is not in the document]
Comment (evcodes) [reply packages/amplify-category-custom/src/utils/constants.ts]: reverted my change. Thanks for pointing this out!

File: .eslint-dictionary.json [hunk 2]
```diff
@@ -50,6 +51,7 @@
   "birthdate",
   "bitwise",
   "bool",
+  "breakcirculardependency",
```
Comment (sdstolworthy) [inline .eslint-dictionary.json:54]: If this is going to be added to the dictionary, please remove the eslint-disable flags in `packages/amplify-category-auth/src/provider-utils/awscloudformation/auth-stack-builder/auth-stack-transform.ts` and `packages/amplify-category-auth/src/provider-utils/awscloudformation/auth-inputs-manager/auth-input-state.ts`

File: .eslint-dictionary.json [hunk 3]
```diff
@@ -257,12 +265,14 @@
...
   "pathname",
   "pathnames",
+  "pbkdf2",
   "pbkdf2sync",
   "pbxproj",
   "pipelined",
   "pipenv",
   "pipfile",
   "pkill",
+  "placeindex",
```
Comment (sdstolworthy) [inline .eslint-dictionary.json:275]: nit: it looks like usages of this misspelled word could be safely renamed in code.

**Review by sdstolworthy — APPROVED** [review]

**Review by goldbez — APPROVED** [review]

File: .eslint-dictionary.json (hunk 3 above)
Comment (evcodes) [reply .eslint-dictionary.json:275]: you're right, I looked into some of the location documentation and that's not a naming convention that has to be all lowercased from what I can tell. Will make the change.

File: packages/amplify-cli-core/API.md [hunk 4]
```diff
@@ -670,7 +670,7 @@ export function generateCustomPoliciesInTemplate(template: Template_2, resourceN
 export const generateOverrideSkeleton: (context: $TSContext, srcResourceDirPath: string, destDirPath: string) => Promise<void>;
 
 // @public (undocumented)
-export const generateTsConfigforProject: (srcResourceDirPath: string, destDirPath: string) => void;
+export const generateTsConfigForProject: (srcResourceDirPath: string, destDirPath: string) => void;
```
[the remark this answers is not in the document]
Comment (evcodes) [reply packages/amplify-cli-core/API.md:673]: reverting this

File: packages/amplify-cli-core/src/overrides-manager/override-skeleton-generator.ts [hunk 5]
```diff
@@ -137,7 +137,7 @@ export const generateAmplifyOverrideProjectBuildFiles = (backendDir: string, src
 /**
  * this method generates the tsconfig file template for overrides
  */
-export const generateTsConfigForProject = (srcResourceDirPath: string, destDirPath: string): void => {
+export const generateTsConfigforProject = (srcResourceDirPath: string, destDirPath: string): void => {
```
Comment (akshbhu) [inline packages/amplify-cli-core/src/overrides-manager/override-skeleton-generator.ts:140]:
```suggestion
export const generateTsConfigForProject = (srcResourceDirPath: string, destDirPath: string): void => {
```

**Review by akshbhu — APPROVED** [review]

File: packages/amplify-cli-core/src/overrides-manager/override-skeleton-generator.ts (hunk 5 above)
Comment (evcodes) [reply packages/amplify-cli-core/src/overrides-manager/override-skeleton-generator.ts:140]: this is the breaking change that @jhockett was referring to above. I am going to keep the old version

**Pull request merged**